manubot/cite/isbn.py

```diff
@@ -0,0 +1,84 @@
+import collections
+import json
+import logging
+import re
+
+
+def get_isbn_citeproc(isbn):
```

Comment:
Are you allowing flexibility to later query multiple ISBN metadata sources?

Reply:
At some point, we may want to have this function fallback to alternative sources.

In addition, this makes it easy to switch whichever function is the default.